src/ucs/type/spinlock.h

```diff
@@ -78,9 +77,24 @@ ucs_recursive_spin_is_owner(const ucs_recursive_spinlock_t *lock,
     return lock->owner == self;
 }
 
+static inline int ucs_spin_try_lock(ucs_spinlock_t *lock)
```

Comment:
minor: `UCS_F_ALWAYS_INLINE`?

Reply:
ah, I guess, we can't use it since `compiler_def.h` can't be included here

Reply:
we can include compiler_def.h - it's installed

Reply:
added force inline